ddtrace/ext/git.py

```diff
@@ -99,6 +99,18 @@ def extract_commit_message(cwd=None):
     raise ValueError(stderr)
 
 
+def extract_workspace_path(cwd=None):
+    # type: (Optional[str]) -> str
+    cmd = subprocess.Popen(
+        ["git", "rev-parse", "--show-toplevel"], stdout=subprocess.PIPE, stderr=subprocess.PIPE, cwd=cwd
+    )
+    stdout, stderr = cmd.communicate()
```

Comment:
Would it make sense to factor this out into a helper for invoking the git CLI binary? I see this pattern is also used in other places.

Reply:
Good catch!